site/client/covid/CovidBars.tsx

```diff
@@ -1,102 +1,20 @@
 import * as React from "react"
-import { computed } from "mobx"
 import { observer } from "mobx-react"
-import { scaleLinear } from "d3-scale"
-import { bind } from "decko"
-
-import { max, keyBy } from "charts/Util"
-
-export enum BarState {
-    highlighted = "highlighted",
-    current = "current",
-    normal = "normal",
-    faint = "faint"
-}
+import { SparkBars } from "charts/SparkBars"
 
 export interface CovidBarsProps<T> {
     data: T[]
     x: (d: T) => number
     y: (d: T) => number | undefined
     xDomain: [number, number]
-    onHover: (d: T | undefined, index: number | undefined) => void
     currentX?: number
     highlightedX?: number
     renderValue?: (d: T | undefined) => JSX.Element | undefined
+    onHover?: (d: T | undefined, index: number | undefined) => void
 }
```

Comment:
Hm, this really is `SparkBarsProps` and should probably be replaced.

Reply:
resolved!